media-sound/jack2/jack2-1.9.11_rc1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads(+)"
+[[ "${PV}" = "2.9999" ]] && inherit git-r3
+inherit eutils python-single-r1 waf-utils multilib-minimal
+
+DESCRIPTION="Jackdmp jack implemention for multi-processor machine"
+HOMEPAGE="http://jackaudio.org/"
+
+RESTRICT="mirror"
+if [[ "${PV}" = "2.9999" ]]; then
+	EGIT_REPO_URI="git://github.com/jackaudio/jack2.git"
+	KEYWORDS=""
+else
+	inherit versionator
+	MY_PV="$(get_version_component_range 1-3)-RC1"
```

Comment:
Had to do this because the tag is uppercase, yet ebuild policy is to prefer lowercase for the ebuilds filename